packages/design-system-react-native/src/components/Text/README.md

```diff
@@ -0,0 +1,214 @@
+# Text
+
+The `Text` component is a customizable text component for React Native applications, leveraging `twrnc` for styling. It allows you to display text with various typography variants, colors, font weights, and styles.
+
+## Props
+
+### `variant`
+
+Optional enum to select between typography variants.
+
+| TYPE                           | REQUIRED | DEFAULT              |
+| :----------------------------- | :------- | :------------------- |
+| [TextVariant](./Text.types.ts) | No       | `TextVariant.BodyMd` |
+
+### `children`
+
+Text content to be displayed.
+
+| TYPE                    | REQUIRED |
+| :---------------------- | :------- |
+| `string` or `ReactNode` | Yes      |
+
+### `color`
+
+Optional prop to set the text color.
+
+| TYPE                         | REQUIRED | DEFAULT                 |
+| :--------------------------- | :------- | :---------------------- |
+| [TextColor](./Text.types.ts) | No       | `TextColor.TextDefault` |
+
+### `fontWeight`
+
+Optional prop to adjust the font weight.
+
+| TYPE                          | REQUIRED | DEFAULT             |
+| :---------------------------- | :------- | :------------------ |
+| [FontWeight](./Text.types.ts) | No       | `FontWeight.Normal` |
+
+### `fontStyle`
+
+Optional prop to adjust the font style.
+
+| TYPE                         | REQUIRED | DEFAULT            |
+| :--------------------------- | :------- | :----------------- |
+| [FontStyle](./Text.types.ts) | No       | `FontStyle.Normal` |
+
+### `twClassNames`
+
+Optional prop to add custom `twrnc` class names for additional styling.
```

Comment:
nit: Should we say tailwind classnames instead of `twrnc`?
```suggestion
Optional prop to add custom Tailwind CSS class names for additional styling.
```